patch/kconfig-exclusions

```diff
@@ -10,3 +10,35 @@ CONFIG_MLXSW_PCI
 [arm64]
 
 [armhf]
+
+[marvell-armhf]
+CONFIG_NO_HZ_IDLE
+CONFIG_HZ_250
```

Comment:
I am sorry, there is still a big misunderstanding going on here. I try to make my point here. If you want 100 Hz as in commit https://github.com/Azure/sonic-linux-kernel/pull/176/commits/3e00dbc2aaf8db8fe7b2450a0674f1529c06a01a you should only need to add it to the *inclusion* file, and the Linux’ Kconfig system will disable the other options (because it’s a choice). Having the “choice option” also in the *exclusion* file should be unnecessary, and only add maintenance effort (if you want to change to 250 Hz, you have to edit several files) and confusing. So, please only add `CONFIG_HZ_100=y`

Reply:
Reverted the changes and we made config to be inline with the generated configuration.